packages/inter-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -197,9 +197,22 @@ const start = async zcf => {
         trace('exiting processTranches loop');
         return;
       }
+      // This await almost certainly does not do what was intended.
+      // `oncePerBlock()` returns an async generator, not a promise,
+      // and does nothing observably effectful to anyone not pulling on
+      // the generator, which this code drops. AFAICT, this code is
+      // equivalent to `await null;`. It imposes a non-delayed turn boundary
+      // and does nothing else.
+      // See https://github.com/Agoric/agoric-sdk/issues/6220
       await oncePerBlock();
 
       // Determine the max allowed tranche size
+      //
+      // This nested await is safe because "terminal-control-flow".
```

Comment:
These are all part of the related design pattern of using for `for await` and an async generator. We should have one comment to that effect and then suppress the overall nested await check in this file